.storybook/preview.js

```diff
@@ -7,7 +7,13 @@ export const decorators = [withFluentProvider, withStrictMode];
 /** @type {import('@storybook/react').Parameters} */
 export const parameters = {
   controls: {
-    disable: true,
+    hideNoControlsWarning: true,
     expanded: true,
   },
+  viewMode: 'docs',
+  docs: {
+    source: {
+      excludeDecorators: true,
```

Comment:
what is this config for ? I'm unable to find any related docs - can you provide those pls?

Reply:
Without this config, the source code for the main story wasn’t being shown properly. 
It’s not in the docs, I found it in this comment: https://github.com/storybookjs/storybook/issues/12596#issuecomment-872754449